### ISSUE TITLE
Can't build on Windows

### ISSUE THREAD

**Issue by darlingtonamz** [IonicaBizau/magnesium#10]
I don't know if it's a Mac only app. If not, I can't build it or use it. If so, please consider a Windows port. MacOS is already flooded with Terminal. Bring it to Windows, and you'll be shocked by the support you'll get

**Comment by IonicaBizau** [comment]
I'd like to have it compatible with Windows. I'm using Linux. For reference, can you post the errors you're getting?

**Comment by darlingtonamz** [comment]
The error is longer, but this will do.
```
npm ERR! Windows_NT 10.0.14316
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Amanze\\AppData\\Local\\npm\\node_modules\\npm\\bin\\npm-cli.js" "i" "-g" "magnesium"
npm ERR! node v4.4.3
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE

npm ERR! pty.js@0.3.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the pty.js@0.3.0 install script 'node-gyp rebuild'.
```

**Comment by Tacolizard** [comment]
Yeah he's getting the same error I get, it's not specific to this app. I don't think it works with this, but on another node project I fixed it by deleting node_modules and reinstalling node and npm.

**Comment by IonicaBizau** [comment]
I guess you will have to set up the C++ building tools on Windows (Visual Studio or so). Probably `pty.js` fails to be compiled. :cry:

**Comment by Odol** [comment]
hi, the pty.js project is discontinued and not run on window plataform you now need use ptyw.js repository.

steps: 

change the repository pty.js for ptyw,js in web-term project and
you must modify  the absolute path reference
to cmd.exe / powershell.exe for example.

ptyw.fork('C:\\Windows\\System32\\cmd.exe', [], {
            name: 'xterm-color-cmd',
            cols: 80,
            rows: 30,
            cwd: process.env.HOME,
            env: process.env
        });
        
Install c++ tools for visual studio 2013 or 2015. 

Go to web-term project folder  -> node_modules/ptyw.js and execute the next sentence in the Developer Command Prompt for Visual Studio:  

**node-gyp rebuild --target=[electron version] --arch=x64 --dist-url=https://atom.io/download/atom-shell**

alternative to compile native node modules is electron-rebuild.

----------------------------------------------------------  o -----------------------------------------------------------

/// tested on:
node: v6.2.0 
electron: v1.2.1 

// compiler
node-gyp: v3.3.1
// pm
npm: v3.8.9

I hope this helps sorry for my bad english jeje bye. :)

**Comment by IonicaBizau** [comment]
@Odol Wowow, that's useful info! I'm glad that somebody takes care of maintaining the fork! 👍 I think we have to fix this in `web-term`, first, and then update the version here.

**Comment by Odol** [comment]
web-term is now flying on window plataforms xD 
![webterm-winos](https://cloud.githubusercontent.com/assets/11588112/16015946/6b76a50a-316e-11e6-9d94-47e51b86f87c.png)

**Comment by IonicaBizau** [comment]
@Odol Wowowow! ✨ That's amazing news!

**Comment by IonicaBizau** [comment]
@Odol Can you make a pull request in web-term?

**Comment by darlingtonamz** [comment]
Web term is working, nice work @Odol. 
Quick question: Where you able to build the Windows version of Magnesium, into an EXE or MSI ?
If so, please try and add it to this Repo's release if @IonicaBizau permits you.
Thanks.

**Comment by IonicaBizau** [comment]
@darlingtonamz That would be great + #3 ✨ I never packed an electron app before, so maybe, you guys can help me. 😁
